client/dashboard/sites/settings-sftp-ssh/sftp-card.tsx

```diff
@@ -0,0 +1,175 @@
+import { DataForm } from '@automattic/dataviews';
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	BaseControl,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteSftpUsersResetPasswordMutation } from '../../app/queries';
+import ClipboardInputControl from './clipboard-input-control';
+import type { SftpUser } from '../../data/types';
+import type { Field } from '@automattic/dataviews';
+
+const SFTP_URL = 'sftp.wp.com';
+const SFTP_PORT = '22';
+
+type SftpCardFormData = {
+	url: string;
+	port: string;
+	username: string;
+	password: string;
+};
+
+export default function SftpCard( {
+	siteSlug,
+	sftpUsers = [],
+}: {
+	siteSlug: string;
+	sftpUsers: SftpUser[];
+} ) {
+	const { username = '', password = '' } = sftpUsers[ 0 ] ?? {};
+	const mutation = useMutation( siteSftpUsersResetPasswordMutation( siteSlug ) );
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+	const formData = {
+		url: SFTP_URL,
+		port: SFTP_PORT,
+		username,
+		password,
+	};
+
+	const handleCopy = ( label: string ) => {
+		createSuccessNotice(
+			sprintf(
+				/* translators: %s is the copied field */
+				__( 'Copied %s to clipboard.' ),
```

Comment:
This should be sentence case if possible. At the moment it can appear as `Copied Connection command to clipboard.`